README.md

```diff
@@ -73,6 +73,12 @@ Apply OVN DaemonSet and Deployment yamls.
 # Create OVN namespace, service accounts, ovnkube-db headless service, configmap, and policies
 kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/ovn-setup.yaml
 
+# Define egressips.k8s.ovn.org CRD
+kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/k8s.ovn.org_egressips.yaml
+
+# Define egressfirewalls.k8s.ovn.org CRD
+kubectl create -f $HOME/work/src/github.com/ovn-org/ovn-kubernetes/dist/yaml/k8s.ovn.org_egressfirewalls.yaml
```

Comment:
I'd say

    # If you plan to use the Egress IPs or EgressFirewall features, create the corresponding CRDs:
    kubectl create -f ...
    kubectl create -f ...

to make it clear (as eventually clarified in the other issue) that this is an optional step

Reply:
@danwinship , thanks for the comments. What about the following 

# Optionally, if you plan to use the Egress IPs or EgressFirewall features, create the corresponding CRDs: 
# create egressips.k8s.ovn.org CRD
kubectl create -f . . .
# create egressfirewalls.k8s.ovn.org CRD
kubectl create -f . . .